base/kubedriver.go

```diff
@@ -0,0 +1,68 @@
+package base
+
+import (
+	"errors"
+
+	// Import to initialize client auth plugins.
+	_ "k8s.io/client-go/plugin/pkg/client/auth"
+
+	"github.com/iter8-tools/iter8/base/log"
+
+	"helm.sh/helm/v3/pkg/cli"
+
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/client-go/dynamic"
+	"k8s.io/client-go/rest"
+)
+
+// KubeDriver embeds Kube configuration, and
+// enables interaction with a Kubernetes cluster through Kube APIs
+type KubeDriver struct {
+	// EnvSettings provides generic Kubernetes options
+	*cli.EnvSettings
+	// RestConfig is REST configuration of a Kubernetes cluster
+	RestConfig *rest.Config
```

Comment:
It would be preferable to not have a `RestConfig` field in the struct, but instead simply have a `restConfig` local variable, wherever it is invoked (only in one place as far as I can see).

Reply:
Among other things, it would enable us to directly embed `base.KubeDriver` into `driver.KubeDriver` (without any extra fields showing up in driver).

Reply:
done